gui/src/app/SamplerOutputView/SamplerOutputView.tsx

```diff
@@ -197,6 +218,29 @@ const prepareCsvText = (draws: number[][], paramNames: string[], drawChainIds: n
     return [['Chain', 'Draw', ...paramNames].join(','), ...lines].join('\n')
 }
 
+const prepareMultipleCsvsText = (draws: number[][], paramNames: string[], drawChainIds: number[], uniqueChainIds: number[]) => {
```

Comment:
Would you mind adding some comments to this function? I’m having a hard time telling the difference between the drawChainIds and the unique ones (or even what size that they would be expected to be)

Reply:
Added comments.